src/CommandingOfficers/Qis.java

```diff
@@ -0,0 +1,245 @@
+package CommandingOfficers;
+
+import java.util.ArrayList;
+import Engine.GamePath;
+import Engine.GamePath.PathNode;
+import Engine.GameScenario;
+import Engine.XYCoord;
+import Engine.GameEvents.GameEventQueue;
+import Engine.GameEvents.MapChangeEvent;
+import Engine.UnitMods.UnitFightStatModifier;
+import Engine.UnitMods.UnitModifier;
+import Engine.UnitMods.UnitMovementModifier;
+import Terrain.Environment;
+import Terrain.Environment.Weathers;
+import Terrain.MapMaster;
+import Terrain.TerrainType;
+import Units.Unit;
+import Units.UnitContext;
+import Units.MoveTypes.MoveType;
+
+public class Qis extends Commander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  public static CommanderInfo getInfo()
+  {
+    return coInfo;
+  }
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Qis");
+      infoPages.add(new InfoPage(
+          "Commander Qis (pronounced 'keese') has spent his adult life fleeing cold weather, and failing to escape it.\n" +
+          "Whether the cause is simply incredible misfortune or some sort of curse is unclear... he is quite curt on the subject.\n" +
+          "What is known is that Qis has become incredibly adept at using the ever-present snow to his advantage."));
+      infoPages.add(new InfoPage(
+          "Passive:\r\n" +
+          "Watery terrain slows your non-naval vehicles, and they lose -1 movement when starting in rain\n" +
+          "All units have perfect movement in snow.\n" +
+          "The path each unit takes on your turn is coated with snow until your next turn."));
+      infoPages.add(new InfoPage(
+          GroovinMovin.NAME+" ("+GroovinMovin.COST+"):\n" +
+          "+"+GroovinMovin.BUFF+" move for all units\n" +
+          "Snow trails left this turn last +1 turn for each space moved so far"));
+      infoPages.add(new InfoPage(
+          TrailOfChill.NAME+" ("+TrailOfChill.COST+"):\n" +
+          "+"+TrailOfChill.BUFF+" move for all units\n" +
+          "All snow on the map will last "+TrailOfChill.DURATION+" turns"));
+      infoPages.add(new InfoPage(
+          "Hit: Deserts, Desserts\n" +
+          "Miss: Showing up first"));
+      infoPages.add(new InfoPage(
+          "Original art credit:\n" +
+          "@Smile ❤#3047 Discord ID 374728627373211651"));
```

Comment:
There doesn't seem to be a way to put comments on non-text files, so I'll put it here.

1) You said you got permission for custom art? This is a recolor. Really not sure how I feel about that.
2) It doesn't line up correctly. When full-screened it probably doesn't matter, but at the default size...
![Screenshot from 2022-01-01 09-45-28](https://user-images.githubusercontent.com/16439262/147854372-7a8b5986-8b27-4f7e-be7b-c3624f33460e.png)
If we want full-body portraits, maybe we need a way to anchor them so they are framed properly at smaller sizes? I dunno.

Reply:
It's not *just* a recolor. But yes, I'm aware it's based on Koal.
Yeah, I think I'll tweak the alignment deal in `presentationFixes` and report back if I do.

Reply:
It's little more than a recolor, and I'd take badly-drawn originals over recolors any day. I know you got permission from this guy to use it, but I'm not sure it's _his_ permission you'd really need, anyway. But I mean, it's your CO I guess.

And changing the face/eyes to focus on the tree? Now you're just trolling.

Reply:
Hmm. I think I can see your point about recolors. This *is* a parody of Koal, though, so leaning into it a little more isn't bad IMO.
The point about permission is probably fair as well.

As for trolling... Yeah, probably guilty as charged.